roles/openshift_health_checker/test/etcd_traffic_test.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+from openshift_checks.etcd_traffic import EtcdTraffic, OpenShiftCheckException
+
+
+@pytest.mark.parametrize('group_names,is_containerized,version,is_active', [
+    (['masters'], False, "3.5", True),
+    (['masters'], True, "3.6", False),
+    (['nodes'], False, "3.6", False),
+    (['nodes'], True, "3.4", False),
+    (['etcd'], False, "3.4", True),
+    (['etcd'], True, "3.5", True),
+    (['etcd'], False, "3.5", True),
+    (['masters', 'nodes'], False, "3.5", True),
+    (['masters', 'etcd'], False, "3.3", False),
+    ([], False, "3.4", False),
+    (['lb'], False, "3.6", False),
+    (['nfs'], False, "3.5", False),
+])
+def test_is_active(group_names, is_containerized, version, is_active):
+    task_vars = dict(
+        group_names=group_names,
+        openshift=dict(
+            common=dict(is_containerized=is_containerized, short_version=version),
+        ),
+    )
+    assert EtcdTraffic.is_active(task_vars=task_vars) == is_active
+
+
+@pytest.mark.parametrize('group_names,matched,failed,master_count,extra_words', [
+    (["masters"], True, False, 1, ["Higher than normal etcd traffic detected"]),
+    (["masters", "etcd"], False, True, 1, ["No log messages matched the expression"]),
+    (["etcd"], False, False, 5, []),
+])
+def test_log_matches_high_traffic_msg(group_names, matched, failed, master_count, extra_words):
```

Comment:
This test makes it hard to change code, because it looks for very specific error messages. Any updates to the message would require changes to two files.